### PR TITLE
[TRAFODION-3127] tochar enhancement

### PR DESCRIPTION
**Pull request by kakaxi3019**

### CODE REVIEW

File: core/sql/exp/exp_datetime.cpp [hunk 1]
```diff
@@ -3294,6 +3338,39 @@ convertMonthToStr(Lng32 value, char *&result, UInt32 width)
...
+    "FRI",
+    "SAT"
+  };
+
+  if (bAbbreviation)
+    strcpy(result, dayofweek_abb[value-1]);
+  else
+    strcpy(result, dayofweek[value-1]);
+  // Update result pointer to point to end of string.
+  result += width;
```
Comment (DaveBirdsall) [inline core/sql/exp/exp_datetime.cpp]: Why width instead of strlen(result)? (This is probably OK; I'm just curious.)

**Comment by DaveBirdsall** [comment]
The manuals should be updated. Perhaps you could write a separate JIRA for that purpose? @liuyu000 might be interested.

**Comment by kakaxi3019** [comment]
Hi, Dave.Thanks for reminding. I have wrote a JIRA TRAFODION-3127.

File: core/sql/exp/exp_datetime.cpp (hunk 1 above)
Comment (kakaxi3019) [reply core/sql/exp/exp_datetime.cpp]: Yes. The string length is always 3 or 9, so i assign the value for width. 
For example 
strlen("SUNDAY")=6, but strlen("WEDNESDAY")=9. 
so the length will always be 9.
But here "SUNDAY   "(with 3 whitespace), so your suggestion is good, thank you!

**Comment by liuyu000** [comment]
@DaveBirdsall Thanks for your remiding, I'll update the SQL Reference Manual 😁

**Pull request merged**